sgx-tools/gen-quote.go

```diff
@@ -66,20 +63,38 @@ For example, generate the quote file according to the given local report file:
 			return fmt.Errorf("Unsupport quote type: %v", quoteType)
 		}
 
+		var quote []byte
+
 		if strings.EqualFold(quoteType, intelsgx.QuoteTypeEpidUnlinkable) || strings.EqualFold(quoteType, intelsgx.QuoteTypeEpidLinkable) {
 			spid := context.String("spid")
 			if spid == "" {
 				return fmt.Errorf("spid can't be empty in both epid for unlinkable and epid for linkable modes")
 			}
 
-			if len(spid) != spidLength*2 {
-				return fmt.Errorf("Spid must be %d-character long", spidLength*2)
+			if len(spid) != sgx.SpidLength*2 {
+				return fmt.Errorf("Spid must be %d-character long", sgx.SpidLength*2)
 			}
-		}
 
-		quote, err := intelsgx.GetQuoteEx(quoteType, report, context.String("spid"))
-		if err != nil {
-			return err
+			p := parseSgxEpidAttester(quoteType, spid)
+			attester, err := registration.NewAttester(sgx.SgxEpid, p)
+			if err != nil {
+				return err
+			}
+
+			quote, err = attester.GetQuote(report)
+			if err != nil {
+				return err
+			}
+		} else {
+			attester, err := registration.NewAttester(sgx.SgxEcdsa, nil)
+			if err != nil {
+				return err
+			}
+
+			quote, err = attester.GetQuote(report)
+			if err != nil {
+				return err
+			}
```

Comment:
`attester.GetQuote()` already becomes the common code. Plz make it common to epid and ecdsa cases.